caster.yml

```diff
@@ -0,0 +1,241 @@
+        --prefix=${FLATPAK_DEST} "scandir>=1.10.0" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/df/f5/9c052db7bd54d0cbf1bc0bb6554362bba1012d03e5888950a4f5c5dadc4e/scandir-1.10.0.tar.gz
+        sha256: 4d4631f6062e658e9007ab3149a9b914f3548cb38bfb021c64f39a025ce578ae
+  - name: python3-pylint
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "pylint" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/1a/28/fca86b40597d383cbc8ca3c1e5f95c3970a509811da74319108ea5c752ed/astroid-3.0.2-py3-none-any.whl
+        sha256: d6e62862355f60e716164082d6b4b041d38e2a8cf1c7cd953ded5108bac8ff5c
+      - type: file
+        url: https://files.pythonhosted.org/packages/f5/3a/74a29b11cf2cdfcd6ba89c0cecd70b37cd1ba7b77978ce611eb7a146a832/dill-0.3.7-py3-none-any.whl
+        sha256: 76b122c08ef4ce2eedcd4d1abd8e641114bfc6c2867f49f3c41facf65bf19f5e
+      - type: file
+        url: https://files.pythonhosted.org/packages/d1/b3/8def84f539e7d2289a02f0524b944b15d7c75dab7628bedf1c4f0992029c/isort-5.13.2-py3-none-any.whl
+        sha256: 8ca5e72a8d85860d5a3fa69b8745237f2939afe12dbf656afbcb47fe72d947a6
+      - type: file
+        url: https://files.pythonhosted.org/packages/27/1a/1f68f9ba0c207934b35b86a8ca3aad8395a3d6dd7921c0686e23853ff5a9/mccabe-0.7.0-py2.py3-none-any.whl
+        sha256: 6c2d30ab6be0e4a46919781807b4f0d834ebdd6c6e3dca0bda5a15f863427b6e
+      - type: file
+        url: https://files.pythonhosted.org/packages/be/53/42fe5eab4a09d251a76d0043e018172db324a23fcdac70f77a551c11f618/platformdirs-4.1.0-py3-none-any.whl
+        sha256: 11c8f37bcca40db96d8144522d925583bdb7a31f7b0e37e3ed4318400a8e2380
+      - type: file
+        url: https://files.pythonhosted.org/packages/0b/59/289c76e3ebdaa2ce9910b366aea437dbc1e0e602a0a513abca757fc4997a/pylint-3.0.3-py3-none-any.whl
+        sha256: 7a1585285aefc5165db81083c3e06363a27448f6b467b3b0f30dbd0ac1f73810
+      - *id001
+  - name: python3-PySide2
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "PySide2" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/ea/58/e5a1aa7417f87987f03f02c0d0ce4bb1112ae9981667107ee9a1560f3607/PySide2-5.13.2-5.13.2-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl
+        sha256: 7c57fe60ed57a3a8b95d9163abca9caa803a1470f29b40bff8ef4103b97a96c8
+      - type: file
+        url: https://files.pythonhosted.org/packages/31/90/7d22091d25492af3c49f47dd255c761b45c1b93e60e164d8007ed2146758/shiboken2-5.13.2-5.13.2-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl
+        sha256: 81fa9b288c6c4b4c91220fcca2002eadb48fc5c3238e8bd88e982e00ffa77c53
```

Comment:
There's no aarch64 wheel here, building on that arch will fail. You can build pyside2 from source

Reply:
But you haven't restricted the build to x86_64, these are being built for both x86_64 and aarch64